### ISSUE TITLE
[Feature Request]: New Video Player UI/UX Issues

### ISSUE THREAD

**Issue by LinuxinaBit** [FreeTubeApp/FreeTube#6216]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

The new video player UI has various issues related to:
1. Staying on screen for far too long when playback is started or resumed
2. Blocking large portions of the video when displayed, on the edges as well as the (enabled by default) play button

Both of these are especially noticeable when the video is paused 

### Proposed Solution

First off, to fix the play button issue, the play button should
1. By default only be permanently on screen before playback first begins (i.e. when autoplay is disabled) and _never_ when the video is just paused. This prevents issues when pausing the video to view content.
2. Be much smaller and have a relatively transparent dark border with a brighter icon in the middle similar to the old FreeTube player, instead of the current large and opaque white circle, assuming this can somehow be changed.
3. It should disappear immediately (or with a very short animation) when playback begins.

A good bonus to implement (if possible) is a nice popup animation like the current YouTube player when the video is paused/resumed.

Similarly, the rest of the UI needs some changes as well:
1. The icons at the bottom of the player are **way too small** and need to be larger (https://lawsofux.com/fittss-law/).
2. The entire UI needs to  _consistently_ stay around for ~800 ms then fade out quickly (~200 ms) when playback begins and the mouse stops moving, instead of sometimes deciding to hang around for 1-2 seconds then very slowly fade out.
3. The large amount of padding at the bottom of the viewport covers up way to much of the video when open, and needs to be made more compact.

### Alternatives Considered

Turning off the play button as mentioned in #6205 is not an ideal solution, as it ignores the very important default user experience *and* does not fix any of the issues related to the other video player UI elements.

Switching back to the old video player would obviously be a regression, so that is not an option here.

### Issue Labels

visual improvement

### Additional Information

Thank you for looking into this.
I realize not all of these changes will be feasible, but it would be great if you would look into fixing at least some of them :)
Thank you for making such a great app!

**Comment by Nobylo** [comment]
The developers of Freetube seem generally happy with the state of the big play button, the consensus from them is that "if you don't like it then turn it off." I would vouch for UI scaling options even if the current scale seems fine to me

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@Nobylo do your research before saying stuff that is clearly incorrect. The button has already made smaller in https://github.com/FreeTubeApp/FreeTube/pull/6002

**Comment by LinuxinaBit** [comment]
> The button has already made smaller in https://github.com/FreeTubeApp/FreeTube/pull/6002

That's great! I must have missed it. Hopefully that comes next release :)
The other issues are still there though, maybe I'll make it a checklist or something...

**Comment by EvilGremlin** [comment]
I think controls overlay is placed too high for no valid reason. 
I quite enjoyed it being out of the way when i watch 16:9 videos on my 16:10 monitor.
Big button in center is whatever, as long as it's optional (i'd change if from checkbox to size slider though, something like 0 to 30% of video height)

**Comment by deepspaceaxolotl** [comment]
Would definitely appreciate moving the video controls to the bottom of the video frame as well as having them reach the left and right edge of the frame (I keep pausing and unpausing the video instead of hitting the button I want, for instance). Aside from that and the delay/not disappearing and scaling issues (I'd appreciate buttons more similar in size to YouTube's, makes for easier targets to hit), it's a great improvement compared to the previous player.

**Comment by KnowledgeMC** [comment]
Just came across FreeTube the other day for the first time and been loving it, EXCEPT for this very issue. Glad to see I am not the only one. 

I'm not sure why the play/pause button in the middle of the frame is so gigantic and opaque. As OP said, it interferes with the ability to pause the video in order to see things in the frame. 

Have searched in the UI/Player settings and did not see an option to change this.

**Comment by rcchan123** [comment]
The gigantic pause icon in the frame is my biggest issue with FreeTube right now, especially as I often pause videos to read what's on-screen or to study its details (like when watching a chess game). Would love to have some options to adjust or even eliminate the icon.

**Comment by LinuxinaBit** [comment]
As I have said before, the “just disabling it” is not a good solution. The default behavior itself should be changed.
Let’s not bicker about what we think the biggest problem is, they’re all UX defects that need to be resolved. This isn’t Wayland :)
Thank you ❤️

**Comment by WilliamKimball0** [comment]
Copying what youtube, which has massive resources for user testing, does seems like the best bet: just hide the pause/play icon after 500ms-1000ms or so. No need to keep the icon visible on top of the video where it's just an annoyance which prevents e.g. seeing something from the video that you paused to see, but now that massive icon is on the way. And making the size smaller isn't really going to resolve the issue but hiding the icon completely. The player state can always be seen from left corner icon and it's pretty obvious otherwise as well.

**Comment by arrowgent** [comment]
the new shaka-player comming from 0.21.3 to 0.22.x
immediately disabled the button.

if perhaps there is a delay to hide the icon, or, a way to change its size from being half of the screen to a more reasonable 100px

thanks for the update but that default setting on update was the very first thing i had to disable.

**Comment by Victor239** [comment]
I'd like if right-clicking the playback speed button would increase it by 0.25 without opening the dropdown menu. I know the previous version allowed for both 1-click change and dropdown menu but I can't recall exactly how that was implemented.

**Comment by 7echLife** [comment]
It is annoying for the devs to "fix" something by changing what isn't broken or not needing to be fixed. The play/pause button needs to be the same old UI and YouTube's button, much smaller and somewhat transparent. The same goes for the other buttons on the player with some merging like YouTube. It shouldn't be changed in the first place. I use FreeTube less because of this, yet Invidious is getting beat down hard by YouTube. Fixing the UI should be one of the highest priorities for the next update. This software is great, and I do not want it to suffer the same way YouTube and other services are due to unnecessary UI changes. If it ain't broke, don't fix it.

**Comment by deepspaceaxolotl** [comment]
@7echLife There were issues with the old player that this change fixed, but it could use some tweaking still, like the stuff mentioned in this issue, mm.

**Comment by sicherist** [comment]
Yes, I agree. The giant Play-button when paused is in the way. For example if you pause a video to read text.
